types/highland/highland-tests.ts

```diff
@@ -243,6 +250,9 @@ fooStream = fooStream.filter((x: Foo) => {
   return bool;
 });
 
+// $ExpectType Stream<Baz>
+fooStream.filter(isBaz);
+
```

Comment:
Could we add one more test case for this?

```js
// $ExpectType Stream<number>
numStream = numStream.filter((n: number) => n > 3);
```

(This was mostly for my own satisfaction that we can use `is` without breaking vanilla predicates that aren't type based)

Reply:
Sure, makes sense